game.py

```diff
@@ -0,0 +1,115 @@
+import chess 
+
+board = chess.Board()
+who_played = 0
+draw = 0
+draw_requested = 0
+
+def game(input, id):
+	global draw_requested
+	global draw
+	global who_played
+	result = ""
+
+	result = game_input(input, board, id)
+	if result:
+		return result
+
+	if who_played == id:
+		return ["Not your turn!!!"]		
+	use_input = input.replace('/', '')
+	uci_move = to_uci(use_input)
+	if uci_move in list_of_moves(board):
+		result = move(use_input, board)
+		who_played = id
+
+	if end_condition(board, id):	
+		result.append(str(end_condition(board, id)))
+	
+	return result
+
+def end_condition(board, id):
+	if board.is_checkmate(): 
+		winner_text = "User" + str(id) + " is the winner!!!" + "\n" + "There is a checkmate. Type '/start/' if you want to start another game!"
+		return str(winner_text)
+	if board.is_stalemate():
+		return "There is a stalemate on the board. Party results as a draw. Type '/start/' if you want to start new game!"
+
+def game_input(input, board, id):
+	global draw_requested
+	global draw
+	global who_played
+
+	if input == "/start/": 
+		return start_game(board)
+
+	if input == "/moves/":
+		return [str(list_of_moves(board))]
+
+	if input == "/board/":
+		return print_board(board)
+	
+	if input == "/draw/" and draw_requested != id:
+		draw_requested = id
+		return handle_draw()
+	elif input == "/draw/" and draw_requested == id:
+		return ["You send the offer, you can't accept it!"]
+
+	if input == "/cancel_draw/":
+		cancle_draw()
+		return ["Draw wasn't accepted"]	
+
```

Comment:
I see you have used too many "-if" statements. They are too cpu-consuming and expensive. Could you consider using a "switch case:" or hashMap?

Reply:
Pyhton doesn't have switches and even if I use dictionaries or hash maps I'll still have to use if statements.

Reply:
That's my bad. You area right seems like the "if...elif...else" sequence in python is optimized enough.